src/data/nat/totient.lean

```diff
@@ -54,35 +54,28 @@ begin
   exact periodic_coprime a,
 end
 
-lemma Ico_filter_coprime_le {a : ℕ} (k n : ℕ) (a_pos : 0 < a) :
-  ((Ico k (k + n)).filter (coprime a)).card ≤ totient a * (n / a + 1) :=
+lemma Ico_add_mul_filter_coprime_le {a : ℕ} (k m : ℕ) (a_pos : 0 < a) :
```

Comment:
This seems strictly less general than the original lemma. Why not prove it as a corollary instead?

Reply:
The new lemma is stronger in the case where `n = m * a`, since the RHS now doesn't include the `+ 1`. Still, I could prove `Ico_filter_coprime_le` as a corollary of this, that's a good TODO.